packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -397,7 +399,7 @@ function VM-Install-From-Zip {
             VM-Install-Shortcut -toolName $toolName -category $category -executablePath $executablePath -consoleApp $consoleApp -arguments $arguments
             Install-BinFile -Name $toolName -Path $executablePath
         }
-        return $executablePath
+        return ,@($toolDir, $executablePath)
```

Comment:
Strange syntax with this comma. Maybe we should add a comment to clarify what this does, but this shouldn't block the PR as the packages are broken.